rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -3628,7 +3650,6 @@ extern "C" rmw_ret_t rmw_service_server_is_available(
   ret =
     common_context->graph_cache.get_writer_count(sub_topic_name, &number_of_response_publishers);
   if (ret != RMW_RET_OK || 0 == number_of_response_publishers) {
-    // error
```

Comment:
@eboasson why this change?

Reply:
The error case was self-evident and the comment was incorrect for the common case (no response publishers discovered yet). But, yeah, perhaps that ought to be a separate commit.